src/html-legend.js

```diff
@@ -16,7 +14,8 @@ dc.htmlLegend = function () {
         _container,
         _legendText = dc.pluck('name'),
         _maxItems,
-        _horizontal = false;
+        _horizontal = false,
+        _legendItemClass;
 
     var _l;
```

Comment:
Do you mean the _legendItemClass variable? This can be overridden using the legendItemClass method in below file. 
The legend highlighting is probably only useful/works for pie chart legends and not for multi line charts/stacked bar charts. 
One thing good may be to add the legend highlight as optional parameter(highlightSelected) allowing user to choose.

Reply:
Sorry I meant the `var _l`

Yes, I also agree that making the highlight selection as optional setting is a good idea. Thanks!

Reply:
will move _l to be a  a local variable and add highlightSelected as optional setting